tests/cancun/eip4844_blobs/test_blob_txs.py

```diff
@@ -892,6 +892,59 @@ def test_sufficient_balance_blob_tx_pre_fund_tx(
     )
 
 
+@pytest.mark.parametrize(
+    "tx_access_list",
+    [[], [AccessList(address=100, storage_keys=[100, 200])]],
+    ids=["no_access_list", "access_list"],
+)
+@pytest.mark.parametrize("tx_max_fee_per_gas", [7, 14])
+@pytest.mark.parametrize("tx_max_priority_fee_per_gas", [0, 7])
+@pytest.mark.parametrize("tx_value", [0, 1])
+@pytest.mark.parametrize(
+    "tx_calldata",
+    [b"", b"\x01"],
+    ids=["no_calldata", "single_zero_calldata"],
+)
+@pytest.mark.parametrize("tx_max_fee_per_blob_gas", [1, 100])
+@pytest.mark.parametrize(
+    "tx_gas", [500_000], ids=[""]
+)  # Increase gas to account for contract code
+@pytest.mark.parametrize("mid_tx_send_amount", [0, 100])
```

Comment:
I think we should remove this parameterization in favor of always sending 100 wei, just to reduce the number of tests this produces.